core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -1175,5 +1175,11 @@ def __configure_opss_secrets(self):
                     zip_entry = archive_file.getOPSSWallet();
                     FileUtils.extractZipFileContent(archive_file, zip_entry, extract_path)
                     self.wlst_helper.setSharedSecretStoreWithPassword(extract_path, opss_secret_password)
-        self.logger.exiting(class_name=self.__class_name, method_name=_method_name)
-        return extract_path
+            else:
```

Comment:
Hard to tell from this diff, but it looks like this "else" is for "if OPSS_SECRETS in domain_info", still inside "if domain_info is not None".  Seems like it should be:
```
if (domain_info is not None) and (OPSS_SECRETS in domain_info):
    ... use values from domain info
else:
    ... check model context
```
Even though the domain_info section should always be present on a create.